tika-parsers/pom.xml

```diff
@@ -969,7 +969,7 @@
                   <pluginExecutionFilter>
                     <groupId>org.codehaus.gmaven</groupId>
                     <artifactId>groovy-maven-plugin</artifactId>
-                    <version>${groovy.maven.version}</version>
+                    <versionRange>${groovy.maven.version}</versionRange>
```

Comment:
Why do we need this?

Reply:
Eclipse failed to build workspace with "version".